src/mono/mono/metadata/threads.c

```diff
@@ -805,6 +805,15 @@ mono_thread_internal_set_priority (MonoInternalThread *internal, MonoThreadPrior
 			break;
 #ifdef SCHED_BATCH
 		case SCHED_BATCH:
+#endif
+#ifdef SCHED_IA
+		case SCHED_IA:
+#endif
+#ifdef SCHED_FSS
+		case SCHED_FSS:
+#endif
+#ifdef SCHED_FX
+		case SCHED_FX:
```

Comment:
illumos and Solaris at least. Is there a way to tell for sure which platform support which flag? The existing `#ifdef SCHED_BATCH` is also without a comment, I guess because it is self-explanatory feature detection.